src/open_company_web/actions.cljs

```diff
@@ -77,12 +77,14 @@
 
 (defmethod dispatcher/action :auth-settings [db [_ body]]
   (if body
```

Comment:
@bago2k4 does this take into account the HTTP status? What if we get a body, but not a 200?